src/badguy/flame.cpp

```diff
@@ -248,5 +256,10 @@ Flame::on_flip(float height)
   BadGuy::on_flip(height);
   FlipLevelTransformer::transform_flip(m_flip);
 }
+void
+Flame::updateColor(){
```

Comment:
All functions have a snake_case naming convention. And like Tobbi said, the brace goes on a new line.
```suggestion
Flame::update_color()
{
```